cassandra/cassandra.yaml.template

```diff
@@ -291,15 +320,15 @@ concurrent_counter_writes: 32
 # the smaller of 1/4 of heap or 512MB.
 # file_cache_size_in_mb: 512
 
-# Total permitted memory to use for memtables. Cassandra will stop
+# Total permitted memory to use for memtables. Cassandra will stop 
```

Comment:
The modifications to this file as just because we are updating from Cassandra 2.2.4 to 2.2.7 and I am bringing in the new changes from there.

These don't have anything to do with the tuning we are currently configuring.